Automattic/jetpack
# CSS Tidy: Add `accent-color`, `aspect-ratio`, `gap`, and `text-underline-offset` properties

Related to: https://github.com/Automattic/wp-calypso/issues/62631

#### Changes proposed in this Pull Request:

Adding the following new CSS properties to CSS Tidy:

- accent-color
- aspect-ratio
- gap 
- text-underline-offset

#### Testing instructions:






1. Go to Appearance → Customize → Additional CSS
2. Add `accent-color`, `aspect-ratio`, `gap`, and `text-underline-offset` to a selector
3. Save changes
4. Refresh to see that properties have not been stripped out

#### Does this pull request change what data or activity we track or use?
No.
